### PR TITLE
Home Page: Create a smooth scrolling home page

### PR DESCRIPTION
**Pull request by shlior7**
https://user-images.githubusercontent.com/34008861/170483429-1e30c1d4-0af5-4970-8225-8dd202fce0ed.mp4

### CODE REVIEW

**Review by ShellyGolden — APPROVED** [review]
Looks good!!

File: templates/home/home.html [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  <a href="#start">How to Start?</a>
+  <a href="#foryou">What's in it for you?</a>
+  <a href="#costs">Costs and price?</a>
+  <a href="#team">Team</a>
+</nav>
+<div class="wrapper">
+  <section id="maincode">
+    <article>
+      <h1>Xpense</h1>
+      <p>Welcome to Xpense! You're a home owner? A financial responsible person? Need some example on financial management? Awesome, you found the right spot to make your dreams come true.</p>
```
Comment (LiorKGOW) [inline templates/home/home.html:16]: I think that : "you *have* found"
Is better suited here

File: templates/home/home.html [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+      <p>Already excited and intersted to hear more? Nice, go on and scroll down.</p>
+    </article>
+  </section>
+  <section id="whatis">
+    <article>
+      <h1>What is Xpense?</h1>
+      <p>The Xpense projects goal is to be an open source platform for personal financial management and improvement by monitoring, sharing and discovering other peoples financial management plans. </p>
+      <p>By comparing your situation of life and financial management you can learn read tips and learn how you can be a better financial manager of your own home and family</p>
```
Comment (LiorKGOW) [inline templates/home/home.html:24]: one learn too much?
maybe: you can read tips and learn....

File: templates/home/home.html [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    </article>
+  </section>
+  <section id="start">
+    <article>
+      <h1>How to start?</h1>
+      <p>So firstly you can take a look at the global page where you can see all the houses that are public and see their average expenses.</p>
```
Comment (LiorKGOW) [inline templates/home/home.html:30]: all *of* the *public houses* suites better here in my opinion

File: templates/home/home.html [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+      <h1>What's in it for you?</h1>
+      <p>You can keep your expenses data and see the data presented with graphs by year and divided monthly, see global houses financial management filtered by your liking.</p>
+      <p>in the tips section you can see tips posted on popular categories and post some of your own.</p>
+    </article>
+  </section>
+  <section id="costs">
+    <article>
+      <h1>Costs and price?</h1>
+      <p>Free as a bird.</p>
+      <p>you can pay us in grades or job offering we accept no money.</p>
```
Comment (LiorKGOW) [inline templates/home/home.html:45]: I wholeheartedly support this statement @shlior7 👍

**Review by LiorKGOW — CHANGES_REQUESTED** [review]
Looks really nice @shlior7 I like it a lot 🥇 
left you some sintax adjustement that I think could fit better on some places, pls check them out 😇 

In addition, Could you add maybe on the video how the users go into the login / see the global page / tips page, etc..
Does this PR include the options of the main navbar of the site?
I haven't seen it on the video

**Review by LiorKGOW — COMMENTED** [review]
overall looks really really nice @shlior7

**Comment by LiorKGOW** [comment]
> > In addition, Could you add maybe on the video how the users go into the login / see the global page / tips page, etc..
> > Does this PR include the options of the main navbar of the site?
> > I haven't seen it on the video
> 
> The Main nav bar where the global, login and everything else is still on top image

Oh OK, because it doesn't show on the clip

File: templates/home/home.html [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+  <a href="#start">How to Start?</a>
+  <a href="#foryou">What's in it for you?</a>
+  <a href="#costs">Costs and price?</a>
+  <a href="#team">Team</a>
+</nav>
+<div class="wrapper">
+  <section id="maincode">
+    <article>
+      <h1>Xpense</h1>
+      <p>Welcome to Xpense! You're a home owner? A financial responsible person? Need some example on financial management? Awesome, you have found the right spot to make your dreams come true.</p>
```
Comment (LiorKGOW) [inline templates/home/home.html:16]: Are you a home owner?
(it's a question)

**Pull request merged**